src/components/Dashboard/FilingHistoryList/StaffFiling.vue

```diff
@@ -2,6 +2,17 @@
   <div v-if="filing" class="staff-filing-details body-2">
     <p>{{filing.notationOrOrder}}</p>
 
+    <template v-if="filing.documents && filing.documents.length > 0">
+      <DocumentsList
+                :filing=filing
+                :loadingOne=loadingOne
+                :loadingAll=loadingAll
+                :loadingOneIndex=loadingOneIndex
+                @downloadOne="downloadOne(...arguments)"
+                @downloadAll="downloadAll($event)"
```

Comment:
Can you attach a screenshot, please?

Reply:
wrong image! Let me update it

Reply:
Looks good!

Reply:
Where is the code that displays "No docs" ?